site/content/integrate/apps/quickstart/quick-start-jwt/_index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "JWT in Go"
+heading: "Use JWT in a Mattermost App in Go"
+description: "This quick start guide will walk you through the basics of using JWT in a Mattermost App in Go."
+weight: 30
+---
+
+This quick start guide will walk you through the basics of using [JSON Web Tokens (JWT)](https://jwt.io/) in a Mattermost App in Go. In this guide you will review an App that:
```

Comment:
I think we should explain why you would want to use a JWT token for App development. It's so the App can validate the legitimacy of the request, to verify that this request is indeed coming from the Mattermost server we are expecting it to come from.

Reply:
Great point, I worked this in!